deps/chakrashim/include/v8.h

```diff
@@ -1585,21 +1583,21 @@ class FunctionCallbackInfo {
   }
 
  private:
+  Value** _args;
```

Comment:
There must be a compiler setting to disable these warnings. Didn't bother to tweak that; re-ordering is easy.